about.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
+    <link rel="stylesheet" href="about.css">
+    <title>Lagos Tech Fest.</title>
+    <script src="https://kit.fontawesome.com/97c3dbf8a8.js" crossorigin="anonymous"></script>
+    <script type="module" src="script.js"></script>
+  </head>
+  <body>
+    <header>
+      <img id="menu" src="images/menu.png" alt="mobile menu" />
+    </header>
+    <!-- desktop menu -->
+    <header id="desktop-menu">
+      <ul class="navigation">
+        <li><img class="facebook" src="images/facebook2.jpeg" alt="facebook" /></li>
+        <li><img class="twitter" src="images/twitter.png" alt="twitter" /></li>
+        <li><p class="nav-desk">English</p></li>
+        <li><p class="nav-desk">My Page</p></li>
+        <li><p class="nav-desk">Logout</p></li>
+      </ul>
+      <nav class="desk-nav">
+        <div class="div-image-desk">
+          <a href="home.html"><img class="image-desk" src="images/Logo.jpg" alt="image"/></a>
+        </div>
+        <div class="container-desk">
+          <ul class="list-desk">
+            <li><a class="anchor" href="about.html">About</a></li>
+            <li><a class="anchor" href="#">Program</a></li>
+            <li><a class="anchor" href="#">join</a></li>
+            <li><a class="anchor" href="#">News</a></li>
+            <li><a class="point" href="#">Sponsor</a></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <section class="container">
+      <div class="introduction">
+        <p class="intro-text">"Hello, Tech world!"</p>
+        <h1>Examining the evolution of E-commerce in Africa</h1>
+        <p class="paragraph-title">
+            The event annually provides an unrivaled platform to identify opportunities and plan strategies by having a strong understanding of the investment opportunities in the country’s tech ecosystem. Driving Investment has been at the core of the event strategy, We provide an opportunity to develop strategies based on the latest regulatory developments.
+        </p>
+        <p class="address">For more enquiries about Lagos Tech Fest.</p>
+        <p class="address-2">lagostechfest@techisfuture.org</p>
+      </div>
+    </section>
```

Comment:
- [ ] Your first section has the elements like the template, but there is a background image missing. Kindly add a background image to differentiate this section from the next one and from the navbar on top. Also remember to add a margin in the box text element of this section.
 - Your design:
 - 
<img width="1027" alt="image" src="https://user-images.githubusercontent.com/61919617/225372772-10214041-1987-4a4a-86ed-b324bb6c60e0.png">

 - Template design:
 - 
<img width="744" alt="image" src="https://user-images.githubusercontent.com/61919617/225372868-bcd51857-1f67-4c34-81ef-3b9d1d2ce3cc.png">